apps/extension/src/background/session/handler.ts

```diff
@@ -0,0 +1,31 @@
+import browser, { Runtime } from "webextension-polyfill";
+import { KeyRingService } from "../keyring";
+import { SessionMsg, SessionMsgType } from "extension";
+
+export class SessionHandler {
```

Comment:
Do we actually need a class here? It seems that the only site where we initiate this is not persisting it, we just end up calling the constructor where we set the listeners.

Reply:
This doesn't necessarily need to be a class.